### DISCUSSION TITLE
Packaging an UWP windows application

### DISCUSSION THREAD

**Discussion by shubhamgupta1313** [unoplatform/uno#10382] [General]
I am trying to develop an UWP windows application build using XAML. It runs perfectly on the local host. For distribution of my application, I created an msix package using visual studio 2022 preview and able to create the package. According to the documentation, After the package gets created, an installer file automatically gets copied in the installer location. But I am not able to find any of the installer file at the installer location. Visual studio terminal shows that app package has been published and installer files get copied but there are no installer files at that location. In the app packages folder of my repo, I am able to find the msix package and the corresponding PowerShell file to run it. I am able to install the application through the PowerShell file but I need to have the .exe files. Any views why this is happening or some alternate approach for packaging?

Attached the screenshots of the terminal showing the successful installation message and PowerShell file and msix package.
![Screenshot (104)](https://user-images.githubusercontent.com/43696690/201019311-70c8d4bc-9334-49b1-891a-e84b23d9bc0b.png)
![Screenshot (105)](https://user-images.githubusercontent.com/43696690/201019322-f7875c0a-ae79-4a00-8e1f-a34c0ad8c686.png)
#UWP #visual studio #msix #packaging

**Comment by jeromelaban** [comment]
WinAppSDK (this is what you're using if you build for `net6.0-windows`) MSIX packaging is meant to be installed as a package, not as an executable you can run.

If you want an unpackaged app, see [this doc](https://platform.uno/docs/articles/features/winapp-sdk-specifics.html?q=unpac#un-packaged-application-support).
